google/resource_bigquery_dataset.go

```diff
@@ -238,6 +238,12 @@ func resourceBigQueryDatasetRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("dataset_id", res.DatasetReference.DatasetId)
 	d.Set("default_table_expiration_ms", res.DefaultTableExpirationMs)
 
+	// Older Tables in BigQuery have no Location set in the API response. We can safely assume that these tables
```

Comment:
This is only going to come up during import, right? If so I wouldn't mind a comment that mentions that just to be extra clear. Also mind moving the other d.Set for location into an else with this clause?